contracts/OffsetHelper.sol

```diff
@@ -280,17 +254,17 @@ contract OffsetHelper is OffsetHelperStorage {
     function withdraw(address _erc20Addr, uint256 _amount) public {
         require(
             balances[msg.sender][_erc20Addr] >= _amount,
-            "You don't have enough to withdraw."
+            "Insufficient balance"
         );
 
-        IERC20(_erc20Addr).transfer(msg.sender, _amount);
+        IERC20(_erc20Addr).safeTransfer(msg.sender, _amount);
         balances[msg.sender][_erc20Addr] -= _amount;
     }
 
     // @description allow people to deposit BCT / NCT
```

Comment:
This comment is not correct anymore.

Reply:
What do you mean?

Reply:
`isRedeemable` allowed the use of only NCT or BCT whereas now `isEligible` allows the use of a bunch of other tokens on top, right? I think this is problematic in general, see https://github.com/ToucanProtocol/example-implementations/pull/25#issuecomment-1147239552